src/components/Layout/ScrollableParent.tsx

```diff
@@ -28,21 +40,31 @@ export function ScrollableParent(props: PropsWithChildren<ScrollableParentContex
     el.style.height = "100%";
     return el;
   }, []);
+  const [tick, setTick] = useState(0);
+  const [hasScrollableContent, setHasScrollableContent] = useState(true);
   const scrollableRef = useRef<HTMLDivElement | null>(null);
   const { paddingLeft, paddingRight, ...otherXss } = xss || {};
-  const context: ScrollableParentContextProps = { scrollableEl, pl: paddingLeft ?? 0, pr: paddingRight ?? 0 };
+  const context: ScrollableParentContextProps = {
+    scrollableEl,
+    pl: paddingLeft ?? 0,
+    pr: paddingRight ?? 0,
+    setPortalTick: setTick,
+  };
 
   useEffect(() => {
     scrollableRef.current!.appendChild(scrollableEl);
   }, [scrollableEl]);
 
+  // Whenever the `tick` changes, reevaluate the needs for fallback scrollable styles
+  useEffect(() => setHasScrollableContent(scrollableEl.childNodes.length > 0), [tick]);
```

Comment:
I wonder if we need another `useState` here for this and instead could just do:

```
const hasScrollableContent = scrollableEl.childNodes.length > 0;
```

?

It seems cheap to compute on each render.

Reply:
Hmm.. Yea, I just kind of went all hook based for this, but it seems even this `useEffect` is not necessary, or the `tick` value. So this would become:
```
const [, setTick] = useState(0);
const hasScrollableContent = scrollableEl.childNodes.length > 0;
```
And that works! Pretty cool... I really figured I'd have to watch that `tick` value change, but I guess just calling a State Hook triggers the re-render.

Reply:
Nice!